### ISSUE TITLE
Add Applications alias to UTM.dmg file

### ISSUE THREAD

**Issue by conath** [utmapp/UTM#3257]
There have been multiple support requests this month because users were unfamiliar with installing Mac apps from disk image. They attended to run UTM from the DMG and it causes issues such as unable to open .utm files. 

We should include a link to `/Applications` in the DMG like many other apps do:

![image](https://user-images.githubusercontent.com/12073163/140607874-805f064c-b389-4406-8d87-a5d7e8e7dc38.jpeg)

**Comment by ktprograms** [comment]
Once I upgrade to Monterey I'm going to look at the .dmg artwork/layout CLI tools. (I want to do it in a VM)

**Comment by semaja2** [comment]
I would suggest implementing the path detection logic, many other Mac apps deploy this as the alias is usually still not enough

Essentially if the app starts outside of /Applications it will offer the user to move it for them

**Comment by conath** [comment]
I've created a background image for use in the UTM.dmg.

![To install UTM, drag the app icon to Applications.](https://user-images.githubusercontent.com/12073163/141318432-5057b017-f911-458b-9bf4-3ebae8b264ce.jpg)

AFAIK the image needs to be in TIFF format so I've attached that here as well. [background.tiff.zip](https://github.com/utmapp/UTM/files/7521197/background.tiff.zip)

(I downloaded the latest Discord.dmg and used their background image as reference, but no pixels from their image are included in mine)

We still need a VolumeIcon.icns set.
